### ISSUE TITLE
Name suggestions for the proposed working group

### ISSUE THREAD

**Issue by pchampin** [solid/solid-wg-charter#73]
Several W3C members objected to the creation of the Solid Working Group, arguing that [the charter](https://www.w3.org/2023/09/proposed-solid-wg-charter.html) seemed too much biased towards a per-conceived solution, rather than trying to solve a problem. See https://github.com/w3c/charter-drafts/issues/458 .

In order to respond to these objections, we intend to submit a new charter where the Solid specifications will be used as input, but where the name and mission of the working group are expressed in more generic terms. The mission has been [updated]() but the name is still to be decided.

Please make proposals for the name of the upcoming Working Group by responding to this issue with the text:

PROPOSAL: &lt;your proposal here&gt;

optionally followed by an explanation/rationale for the name you propose. You may also respond with comments on previous proposals.

**Comment by elf-pavlik** [comment]
PROPOSAL: **Protected Data Platform** or **Protected Linked Data Platform**

Term *Protected Resource* is an established term used, for example, across [OAuth2 family of specs](https://www.ietf.org/archive/id/draft-ietf-oauth-v2-1-10.html#name-role). Data Platform takes inspiration from [Linked Data Platform](https://www.w3.org/TR/ldp/)

-----

Side note: If we convert this issue into a GitHub discussion, we will have a threaded conversation and an upvote feature.

**Comment by csarven** [comment]
PROPOSAL: Socially-aware Web WG

It is a bit TimBL speak and fun and kind of meaningful and kind of not...

PROPOSAL: Personal and Social Web WG

Or something that hits "personal" and/or "social".

PROPOSAL: Personal Storage WG

PROPOSAL: Control Yourself WG

PROPOSAL: Fixing the Web WG

PROPOSAL: Enquire Within WG

**Comment by elf-pavlik** [comment]
:-1: To using term *social*. When it comes to social media, people tend to think about microblogging and dumb social networks like Twitter, TikTok, and Instagram, which are often accused of messing up the minds of the youth.

:+1: To using terms like *collaborative* and *online collaboration* instead. Which, to me, it covers current silos like Wikipedia, Github, MS Teams, Asana, Jira, you name it. It also includes services provided by popular open-source platforms like GitLab, Discourse, RocketChat, OpenFoodNetwork, you name it again.

**Comment by csarven** [comment]
@elf-pavlik Perhaps but is this issue for just suggestions or are we collecting thoughts as well? Because I don't think "Linked Data" in the name of a WG is going to fly =)

**Comment by melvincarvalho** [comment]
PROPOSAL: **Social Linked Data WG**

Was the original name of Solid. It implies an identity element.  And in weaving the web, tim writes, "The Web is more a Social innovation than a technical one".  Social implies user generated content, which in turn is stored with different privacy gradients.  The heavy emphasis on Linked Data as a solution is included too.  And it can be seen as an upgrade of LDP to include social.  If LDP is a REC, then Social LD(P) could be too.

**Comment by michielbdejong** [comment]
PROPOSAL: Personal Data Store WG
PROPOSAL: Personal Linked Data Store WG

**Comment by michielbdejong** [comment]
PROPOSAL: Bring Your Own Storage WG
PROPOSAL: Remote Storage WG

**Comment by melvincarvalho** [comment]
> PROPOSAL: Remote Storage WG

This might be an idea, the RS community is still active I believe, and has some good devs.  Some could be folded in.

**Comment by michielbdejong** [comment]
PROPOSAL: Personal Online Datastore WG

**Comment by TallTed** [comment]
**PROPOSAL:** Socially Integrating Data and Apps WG (SIDAWG) (pronounced sea-dawg)

_My_ photos, posts, comments, etc., live in _my_ storage space (sometimes called a pod), and SIDAWG apps access all that data in _my_ storage space as well as in the storage spaces of my contacts, in order to present a unified view of _our_ photos, conversations, etc.

Browsers display ads (no matter how many blockers you have), so SIDAWG apps (which often run within browsers) can also display ads, providing a revenue stream for the app authors, and thus incentivizing some shift from the monolithic social media apps/sites.

**Comment by CxRes** [comment]
**PROPOSAL**: Semantically Linked Decentralized Data WG
**PROPOSAL**: Shared Linked Decentralized Data WG
 
(still pronounced solid or maybe sled)

Seems closest to the mission!

**Comment by CxRes** [comment]
PROPOSAL: Samaritan Owned Linked Data WG (still solid)

**Comment by CxRes** [comment]
PROPOSAL: Samaritan Owned Linked Information Depository WG (actually SOLID)!

**Comment by CxRes** [comment]
I prefer not to use "Personal" or "Social" as descriptors of Data in Solid (ahem PUMPKIN) is for all kinds of data as I [observed](https://github.com/solid/specification/blob/ea73324a7ec50a7ddf9e83ef5be2614962a5aa6c/meetings/2024-03-06.md?plain=1#L78) and @timbl re-enforced in CG meeting on 2024-03-06.

**Comment by TallTed** [comment]
"Samaritan" doesn't seem appropriate, here.

PROPOSAL: **Semantically Integrating Distributed Data and Apps WG (SIDDAWG)** (pronounced sea-dawg)

Mostly the same as [above](https://github.com/solid/solid-wg-charter/issues/73#issuecomment-2023805598)...

**Comment by bourgeoa** [comment]
PROPOSAL : Linked HTTP storage

**Comment by elf-pavlik** [comment]
:-1: to *personal*, I see data spaces for groups/organizations as one of the key features of solid. A big part of the complexity comes with that compared to data space fully controlled by a single person. We are collaborating here using @solid organization rather than someone's personal space.

**Comment by elf-pavlik** [comment]
PROPOSAL: **Sovereign Secure Data Spaces**

**Comment by cwilso** [comment]
As I've given strong direct feedback on this topic, I wanted to share it in this thread:

Your working group name should capture what space your working group is trying to solve problems in; as a new area, this is especially important.  I would suggest that it's an anti-pattern to try to backronym it into something cute like SIDAWG (sorry, Ted, to pick on a suggestion from you; it's not the only one) because "Semantically Integrating Data and Apps" seems like not a 1:1 alignment with your charter, nor is it instantly understandable.  "Personal Data Storage WG" is understandable; "SOLID WG" is not (unless you already know what the SOLID protocol is).  

(I will also point out that this is about the WG name to me; your spec may still be called SOLID or PUMPKIN or whatever else you want, because you're down in the details then.)

**Comment by CxRes** [comment]
Bouncing off @elf-pavlik idea, how about just:

**PROPOSAL**: Sovereign Linked Data WG

or even,

**PROPOSAL** Sovereign Linked Information Depository WG

**Comment by CxRes** [comment]
If @TallTed does not prefer Samaritan,

**PROPOSAL**: Self-owned Linked Data WG

or,

PROPOSAL Self-owned Linked Information Depository WG

I think it really captures what we are doing!

**Comment by melvincarvalho** [comment]
> Self-owned

Can also have negative connotations such as "self own".  Though playful and amusing.

https://dictionary.cambridge.org/dictionary/english/self-own

**Comment by balessan** [comment]
> 👎 to personal, I see data spaces for groups/organizations as one of the key features of solid.

> I prefer not to use "Personal" or "Social" as descriptors of Data in Solid (ahem PUMPKIN) is for all kinds of data as I [observed](https://github.com/solid/specification/blob/ea73324a7ec50a7ddf9e83ef5be2614962a5aa6c/meetings/2024-03-06.md?plain=1#L78) and @timbl re-enforced in CG meeting on 2024-03-06.

I am also in favor of avoiding "Personal" or "Social" as we, at Startin'blox are involved in a few European data spaces where we see an interest for solid as a protocol enabling efficient data sharing between organisations.

**Comment by bourgeoa** [comment]
PROPOSAL : Linked Web Storage

**Comment by TallTed** [comment]
But it's about much more than storage...

**Comment by cwilso** [comment]
@TallTed can you articulate what more it is about that is critical to put in?

**Comment by bourgeoa** [comment]
> But it's about much more than storage...

On the protocol itself or on the general SOLID vision ?

On the protocol I see it as a RW web storage specification that use :
- web protocols to access all stored elements (any data type)
  - web URIs
  - HTTP protocol (Headers, web operations ...)
- Linked data documents to :
  - set/discover relations between stored elements (container/resource)
  - authorize access to container/resource
  - identify Agents (storage owners, client users)

On top of the protocol and for the Vision you need at least some 
- authentication system for Agents (Identity provider)
- interaction between agents through messaging systems

**Comment by TallTed** [comment]
I think the protocol(s) (the names of which may or may not include any reference to "Solid") must take into account that there may be multiple users with their own identifiers (and even multiple identifiers per user), multiple storages (potentially multiple storages per user), and multiple apps reading from and writing to those storages. All of these must be loosely coupled, such that any given app can discover, read from, and write to all relevant storages for all users involved in a given network, whether that's business, community, family, or otherwise.

This is about decentralization and democratization of the Web.

For example, I want to be able to see a photo album, constructed *on the fly*, that includes all the pictures saved to all the storage locations for myself and all the people I link to as family, taken on a particular date or at a particular location, a la Flickr and its [compatriots](https://www.google.com/search?q=alternatives+to+flickr).

For example, I want to see (and be able to join!) a dynamically assembled, threaded conversation, starting with an initial post, with responses to that post, and responses to those responses, etc., a la Reddit, LiveJournal, Dreamwidth.

I could spend a lot of time coming up with more user stories, but I think these few should be enough to drive development of the initial protocol(s).

**Comment by CxRes** [comment]
Reading @TallTed's comment (though this is very broad), how about:

**PROPOSAL**:  Decentralization and Democratization WG (D&D WG)

**Comment by megoth** [comment]
I do like the idea or reclaiming the decentralized term. And it opens for the cheeky:

PROPOSAL: Decentralized Online Storage (the DOS protocol 😅)

I first thought of it in jest, but at the same I kinda like it =p In any case, now it's out there.

There's also the longer:

PROPOSAL: Decentralized Online Storage & Identity (the Dosanti protocol? 🤔)

If anything, I hope these proposals bring some joy into this thread ^_^

**Comment by melvincarvalho** [comment]
> PROPOSAL: Personal Data Store WG PROPOSAL: Personal Linked Data Store WG

@michielbdejong given that you were one of the two objections to the group being named Solid, could you give some indication of which of these names you would find be acceptable?  I do agree that "Solid" itself has become coupled with something specific, namely the solid protocol 0.10.  Would you find loosening it to the more generic "Social Linked Data" work for you.  Tantek eventually accepted that in the Social Web WG, as it was back then, after some discussion and Andrei creating the first spec.

**Comment by bourgeoa** [comment]
,👎 for decentralisation, it is splitting things and says nothing about interoperability with data relations or sharing data structures, which is the purpose of linked data in typeIndex documents.

Is typeIndex part of the actual specification? Is there a need for a kind of shex/shacl shared collection ?

**Comment by bourgeoa** [comment]
@TallTed I have updated https://github.com/solid/solid-wg-charter/issues/73#issuecomment-2027438080 to better take in consideration your expectations on decentralisation and democratisation, these are political issues that could be expressed as interoperability via linked data and linked data structure discovery.

**Comment by melvincarvalho** [comment]
> for decentralisation, it is splitting things and says nothing about interoperability with data relations or sharing data structures, which is the purpose of linked data in typeIndex documents.

It's primarily a federated model

**Comment by CxRes** [comment]
@bourgeoa You cannot look at decentralization in isolation. It must go hand in hand with democratization (which is all about the shared consensus i.e. linked data, interop etc.). Solid specification proper leans more towards addressing decentralization concerns (though it deals with some democratization concerns) whereas client-client specs lean more towards democratization concerns (though they deal depend on decentralization).

I don't know why, but I am liking **Dungeons and Dragons** even more now!

**Comment by melvincarvalho** [comment]
I thought I'd add this, though it's not a personal suggestion, might be some food for thought.  A paper, that I came across, presented at www '23:

SISSI: An Architecture for Semantic Interoperable Self-Sovereign Identity-based Access Control on the Web

https://dl.acm.org/doi/abs/10.1145/3543507.3583409

**Comment by CxRes** [comment]
Sorry, but I absolutely could not resist! Be very inspired:

https://www.youtube.com/watch?v=iah-VBDeoE8

Any charter for D&D WG must begin with this speech!

**Comment by ebremer** [comment]
PROPOSAL : Distributed Apps, Identities, and Storages (DAIS - A dais or daïs is a raised platform at the front of a room or hall, usually for one or more speakers or honored guests.)

PROPOSAL: Apps, Multi-identity, and Authentication for Distributed Environments of URL/URI Storages (AMADEUS)

**Comment by michielbdejong** [comment]
Here is the alphabetical list of 32 names that were suggested:
```
Apps, Multi-identity, and Authentication for Distributed Environments of URL/URI Storages (AMADEUS)
Bring Your Own Storage WG
Control Yourself WG
Decentralization and Democratization WG (D&D WG)
Decentralized Online Storage (the DOS protocol 😅)
Decentralized Online Storage & Identity (the Dosanti protocol? 🤔)
Distributed Apps, Identities, and Storages (DAIS - A dais or daïs is a raised platform at the front of a room or hall)
Enquire Within WG
Fixing the Web WG
Linked HTTP storage
Linked Web Storage
Protected Data Platform
Personal Data Store WG
Protected Linked Data Platform
Personal Linked Data Store WG
Personal Storage WG
Personal Online Datastore WG
Remote Storage WG
Samaritan Owned Linked Data WG (still solid)
Samaritan Owned Linked Information Depository WG (actually SOLID)
Self-owned Linked Data WG
Self-owned Linked Information Depository WG
Semantically Integrating Data and Apps WG (SIDAWG) (pronounced sea-dawg)
Semantically Linked Decentralized Data WG
Shared Linked Decentralized Data WG
Semantically Integrating Distributed Data and Apps WG (SIDDAWG) (pronounced sea-dawg)
Social and Personal Data WG
Social Linked Data WG
Socially-aware Storage WG
Sovereign Linked Data WG
Sovereign Secure Data Spaces
Sovereign Secure Linked Data Spaces
```
As discussed in the CG meeting today I'll set up a topic where we can thumbs-up and thumbs-down to see which one resonates most. Will post the link here once I've set it up.

**Comment by gatemezing** [comment]
+1 to "Sovereign Linked Data WG"

**Comment by melvincarvalho** [comment]
> As discussed in the CG meeting today I'll set up a topic where we can thumbs-up and thumbs-down to see which one resonates most.

Would it be possible to take into account the votes already cast on this thread?

**Comment by CxRes** [comment]
I would rather these be sorted in the order they were suggested (which is close enough to random or just be randomized) than by another characteristic, as that is likely to affect the decision in subtle ways.

**Comment by elf-pavlik** [comment]
@CxRes to late for that, the issue is already done and AFAIK there is no way to reorder comments https://github.com/solid/solid-wg-charter/issues/75

**Comment by CxRes** [comment]
> @CxRes it is too late for that, the issue is already done, and AFAIK, there is no way to reorder comments #75 This mostly gathers feedback; we will still pick one CG recommendation during one of the CG weekly meetings.

I wish mine was posted with the Silicon Valley video thumbnail as well, then it would be 🥇. :-P

**Comment by TallTed** [comment]
@michielbdejong -- Might be good to propose a closing date, or at least a "we will discuss this on the xyz call starting yyyy-mm-ddZhh:mm", as I think that more people have participated in the suggesting than in the up/down polling.

**Comment by TallTed** [comment]
@pchampin set up a [page that summarizes the votes](https://perso.liris.cnrs.fr/pierre-antoine.champin/2024/solid-name.html) found on #75

**Comment by oolivo** [comment]
Hey all, 

It's unclear to me why we actually need a new name? I understand that 2-3 people objected to the name. But I thought that's why a council was formed to review the objections and determine which ones should be sustained and which ones should be overruled. 

It feels like we're making concessions to try and reduce the number of objections for the council to review, but this is a pretty big (and contentious) one. Wouldn't we want to Council to first determine whether the name needs changing or not in the first place?  Is there zero possibility that we can just keep the original name at this point?  

Most of the names on this list may actually trigger additional objections from other technologies and groups. There are plenty of efforts working on decentralized storage or PDS's. Solid is much more than that. And we may be picking an argument or debate with technologies that are trying to solve that specific problem space. When someone suggested naming the group the PDS WG it raised several subsequent objections from organizations working on PDS technologies. I worry that this will trigger a similar response if we choose a triggering name. So why are we choosing a new name if it's not actually clear that we have to yet? Shouldn't we just wait for the council to deliberate on this? Even members of the council have expressed confusion as to why this isn't just called the Solid charter, so I suspect we may have support for just keeping the name as is. Am I missing something obvious?

**Comment by cwilso** [comment]
I believe the general expectation is that the Council will likely just return the charter to the Team to rework and bring it to a new AC vote.  In general, Councils are not supposed to do detailed adjudication and directions to repair for sub-issues.

I'm not sure why members of the council would have expressed confusion as to why this isn't called the Solid charter, since the council was formed for something CALLED the "Solid WG Charter". (You know my feelings on the matter, of course.)

**Comment by timbl** [comment]
The Council can also override the Formal Objections and let it go though without further AC review.  This may certainly happen if the charter proposal has been modified to take into account the sorts of issue the objectors  brought up.

**Comment by cwilso** [comment]
It is technically true that the Council could override the Formal Objections and let it go through without further AC review.  That was not the Team's recommendation, but it could possibly happen - but I think it's probable that if that were so, it would have to be the charter that was submitted, not a revised one (or the revised one would have to be approved by all those who voted or explicitly abstained on the original charter vote).  There are only three paths, though: 1) The Council can broker consensus via changes to the charter that would remove the objections; 2) The Council upholds the objections (aka the charter fails), though in this case they can recommend mitigations, or 3) the Council overrules the objection - in this last case, there are no required changes (e.g. it cannot require the changes to the charter that have already been suggested.)

(relevent Process: https://www.w3.org/2023/Process-20231103/#council. As per https://www.w3.org/2023/Process-20231103/#CharterReview, I would expect any substantive changes to trigger at least an approval review request from those in the AC who originally voted on the charter; that's certainly the norm.)

I would suggest that such an override is unlikely in this case, for reasons I can't discuss outside the council+Team, but I could be wrong.

As to the naming, your #justsayin point is a red herring.  The charter proposal _for which the Council was formed_ was named "SOLID WG Charter", and those who asked for the group to be renamed did so because _**TAG and AB members are asked to be on a bunch of different councils**_ - and just having the groups named by what month they started in makes it hard to keep them straight.  It is CERTAINLY no indicator that multiple council members "think this work should be called Solid" - just that we need to keep the different councils and their mailing lists straight.

**Comment by VirginiaBalseiro** [comment]
As discussed in [CG meeting](https://github.com/solid/specification/pull/646), we will decide on a final name at the weekly CG meeting next Wednesday April 24th at 14:00 UTC, so make sure to vote on #75 before that time.

**Comment by TallTed** [comment]
@timbl --
> In related news in the mailing list https://lists.w3.org/Archives/Group/group-council-2024-02/2024Feb/0010.html

Please note that the message [cited above](https://github.com/solid/solid-wg-charter/issues/73#issuecomment-2062207287) is not world-readable. Most people on this GitHub thread do not have relevant permissions, and are thus left with incomplete and inaccurate context, based on your paraphrase of what was said there.

@cwilso -- Thank you for [the comment](https://github.com/solid/solid-wg-charter/issues/73#issuecomment-2062410534) explaining the council renaming request. It seems entirely reasonable and appropriate to me, and I would suggest to whomever is in charge of naming the Council mailing lists and similar that they adopt a modified pattern like `group-council-YYYY-MM-<distinct>`, where `<distinct>` is replaced by an string that's unique per council originating within that month, which might or might not be `solid` in this case.

**Comment by timbl** [comment]
> there was an AC meeting. People from the TAG were there as well as the people from the council. They expressed interest in seeing the new charter, especially if it addresses formal objections. If the formal objections are removed, it won't need another AC vote. We should send this new charter to the TAG as soon as possible. Even if the objections are not withdrawn, the council has the power to override them. I would like to send out new charter by the end of this week. I believe we addressed the objections as well as we could.

  @pchampin , https://hackmd.io/cTf4AprtTTi2kjF6NJtAYQ?view

**Comment by timbl** [comment]
@TallTed  Yes I left a reference to a non-public thing, as @cwilso does I suppose when he says "I would suggest that such an override is unlikely in this case, for reasons I can't discuss outside the council+Team, but I could be wrong.".

**Comment by cwilso** [comment]
Tim, the pointer you sent was to a confidential communication of the Council itself.  Please do not share the contents of confidential communications of Councils. (https://www.w3.org/2023/Process-20231103/#council-deliberations)

Actually, part of my reasoning I *CAN* discuss outside the Council: there are three potential resolutions here - either 

1. the Council performs further attempts to broker consensus (i.e. get the formal objectors to remove their objections), or
2. the Council has consensus, OR fails to reach consensus but votes and has a simple majority to uphold the objection (in which case, the Council may provide recommendations on mitigations), or
3. the Council has consensus, OR fails to reach consensus but votes and has a simple majority to override the objections.  If this happens, the Council may make recommendations, but they are not binding (e.g. "you have to take the charter edits since the AC vote"); they are flat-out overruling the objections.  This is the case that I suggested was unlikely; not because I have insider knowledge, but because it is a high bar.  Again, this is merely my opinion.

**Comment by csarven** [comment]
tl;dr: Renaming is not only necessary, it shows genuine efforts made towards incorporating the feedback received, as opposed to ignoring the feedback and hoping that, by chance, the council and other teams will look the other way.

---

Background/Explanation:

The feedback from AC was summarised (and some linked to) in the form of publicly accessible issues since mid 2023-10:

https://github.com/w3c/charter-drafts/issues?q=is%3Aissue+is%3Aopen+wg%2Fsolid

Also summarised in a Solid CG meeting:

https://github.com/solid/specification/blob/main/meetings/2023-10-11.md#wg-charter-proposal-status-update

There is also the TAG review since 2023-11:

https://github.com/w3c/strategy/issues/377#issuecomment-1790209071

As well as AB feedback: [citation needed].

All touch on concerns, including (formal) objections, literally about the naming of the WG. In a nutshell, the name and charter come across as technology/solution/proprietary-centric in its totality. Although naming was just one aspect, the Solid Protocol is even presented as one possible input document for the charter deliverable. We've discussed these topics in this repo as well as Solid CG meetings which are on record.

The Solid CG's position from the get-go has been to take all that feedback and work with the Team contact (Pierre-Antoine) towards a solution addressing the raised concerns as best as it could and to go from there. Of course, certain W3C (Team) decisions are beyond the Solid CG's jurisdiction, and objections touching on those matters fall outside of the Solid CG's jurisdiction.

As I see it, ignoring feedback (including the renaming) and support to update the charter would be both illogical and undermine all the work we've done. It would also be overly hopeful (for my taste) to send something off to the assigned council and expect that they may overlook or hypothetically override, which is counter to all the signs that are given to us.

That's why we actually need a new name.

**Comment by timbl** [comment]
@cwilso said

> Tim, the pointer you sent was to a confidential communication of the Council itself. Please do not share the contents of confidential communications of Councils. (https://www.w3.org/2023/Process-20231103/#council-deliberations)

Well giving the URI of a confidential communication is not the same as giving the communication itself,  but I will delete my message just to be sure.

**Comment by oolivo** [comment]
I'm not at all proposing we ignore the feedback. There were objections about a number of things and many of them have been addressed. This however is one area where in trying to address it, we're likely to only create more confusion and instigate more objections. In the feedback you cite, the name is often pointed to as an issue in the scope and remit of the group not being clear. I.e. not setting a problem statement but rather just saying we're going to standardize Solid. And the name itself is not self descriptive so as a result it comes off as solution centric. Since then, a lot of work has gone in to make the problem statement and scope much clearer in the charter.

So we can still change the name to fully satisfy the objections, and potentially trigger a handful of new objections to be raised due to the new name (I'd be pleasantly surprised if this wasn't the case).  Or we can point to the heavy clarifications that have been made in scope and problem statement, and see if this satisfies most if not all of the objections around it not being obvious what the goals of the group are.

I guess what I'm missing is the W3C making an authoritative statement that all WG's should be named after the problem space they are solving and therefore Solid WG is not an acceptable name . If that's the case, then renaming makes perfect sense and the objection is valid. But if there is no such authoritative statement, and we're making such a major change that is likely to introduce additional complexity and confusion in the space, as well as likely raise additional objections and lose all brand recognition to newcomers, then I just don't get why we would do it to address an objection that is in many ways already addressed by the updates to the charters scope. Changing the name effectively decouples the WG from this CG as well as from all the work that has led up to this proposal. Are we planning on also renaming the CG?

I would like to know the W3C's official stance on this before we make a decision and determine this is necessary. This introduces a lot of noise that I've yet to be convinced or seen compelling evidence is required to move forward. But that doesn't mean it's not out there, but there are a lot of WGs currently in existence that describe the solution being worked on, not the problem space so it's unclear at best.

**Comment by csarven** [comment]
We are taking the feedback and finding ways to improve the situation so that when the updated charter is revisited, there is a sense of mutual understanding among different parties that there was an attempt to address the issues in good faith.

FWIW, have a look over active groups at https://www.w3.org/groups/wg/ or dig deeper into closed WGs or even other CG names. Yes, they mainly focus on problem areas. So are the names that are proposed in this issue. "Solid WG" is nothing like those. Even our earlier use of "Social Linked Data" (SoLiD) would be a more meaningful WG name than "Solid". Having been on Solid since 2015, I can safely say that the term meant absolutely nothing to anyone who didn't have the background or context already, besides a state of matter. If anything, it overlaps with several projects that are also using the term "solid".

As already mentioned, insisting on "Solid WG" works against the feedback. All things being equal, it is safer to go with a different name. We are essentially removing what was once the target of an objection. That's a way to reduce potential objections and tends to hold true the same way for working through the W3C Recommendation Track. The name of the WG is certainly not the hill to die on. Do we want to "fix the Web" (TM) or get stuck on naming / worry about marketing or whatever?

Unfortunately, you'll have to read between the lines to see the "authoritative statements." But as I see it, that's already the feedback from the W3C Community to some degree, in addition to what one draws from various guidance, e.g., https://www.w3.org/Guide/standards-track/#criteria . I'm sure in due time there'll be a crystal clear answer to such questions (if not already out there) but perhaps in the meantime consider creating an issue at https://github.com/w3c/charter-drafts .

The suggested WG names and poll results will be handed off to W3C Team. They or the W3C Community can respond back with what works or doesn't. We ought to still follow W3C community's guidance on this because that's just how this stuff rolls. If the new name becomes an issue, we'll cross that bridge then.

Aside: I have no horse in the naming race. In fact, I chose to abstain from voting on the name and will be content with the consensus, whether it stays as "Solid" or changes to something else.

**Comment by oolivo** [comment]
Thanks @csarven, appreciate the thoughts. Perhaps we should take this conversation to another thread. I'm not trying to be disruptive at the 11th hour or be non-constructive. There is simply a lack of clarity in this whole process that is somewhat frustrating. 

If I'm being candid, the reason I'm coming in at "the 11th hour" is because I was standing by the sidelines waiting for official guidance from the W3C on what we should address vs. shouldn't. It's perfectly reasonable for the group to discuss how we address all objections, if possible. But it's also reasonable for us to agree to disagree on some and receive guidance from the W3C on where to go from there.

The idea in this issue is that "if we address all objections in the revised charter then it will be accepted" seems fraught. Some objections are clear and obvious and simple to address. But there are a number of objections that we simply are not addressing. Some objections were that we should include blockchain in the scope, for example.  We clearly are not addressing that objection. Is that going to be a problem? If so, who decides whether those objections are valid or not? Whats the criteria for which objections need addressing and which ones don't and merit holding the line on? And who's giving that guidance?

I realize I'm asking a larger question, outside the scope of this issue so I'll take it elsewhere.  But I'd like clarity on where the guidance is coming from and whether it is transparant and available somewhere. I hear you that on this particular issue it seems like that may be the direction the W3C is moving in and "I need to read between the lines", but that shouldn't be the case. These things should be clear so that we aren't wasting the groups time and so that folks can better judge where to spend their time. 

I've been closely following all the CG meetings and minutes and issues. And nowhere have I seen clear guidance on which objections need addressing vs. which don't and what the W3C's expectations are for approval. If agreements have been made behind closed doors, I guess thats fine since theres politics involved, but it feels awfully opaque for a process that should be open and transparant. It all feels a bit backwards that a policy that doesn't actually exist today is being used to object and block a charter, and that the fact that it successfully did so will be used as a precedent to essentially will the policy into existence. The W3C should just make the policy and then call it a day. Or not.

I hear your point that we should just move forward and try to compromise as much as possible (within reason) to get the group formed. Ultimately I agree with that. So I'm not so much objecting to the name change as much as I'm asking for clarification. But I realize this probably isn't the place to do it at this point as i'm asking a much more meta question. 

Thanks.

**Comment by cwilso** [comment]
@oolivo "The W3C" cannot give definitive guidance on what _**must**_ be addressed.  I would say you _should_ try to address all feedback.  Ultimately, the "must" will be determined by either 1) a lack of Formal Objections from the AC, or 2) a Formal Objection Council.  There is no single arbiter of decisions.  The easiest path is to address all the feedback in all FOs received; if the Team or Council can broker all formal objectors removing their FOs, you would be done.

Of course, that may not be possible, or may not be something you want to do.  I don't see any request that blockchain be in the scope in the AC poll; but if there were something really antithetical - or something that was a clear "if we fix this FO, we're trading for this other FO" - then it would probably ultimately go to a FO Council to adjudicate.

(In this case, I would point out there were eight different comments in the charter response citing the focus on SOLID in the name or the choice of SOLID as the only possible solution, including all four Formal objections.   That does in fact seem like something that should be addressed.)

**Comment by elf-pavlik** [comment]
I am speaking from the perspective of a regular Solid CG participant. To my understanding, the W3C Team is proposing a new working group, and Solid CG provides feedback to our best capacity. Given that, I trust @pchampin to lead this process and incorporate Solid CG feedback in the best possible way.

Wherever the W3C process appears unclear, maybe we should raise it in https://github.com/w3c/w3process/issues/. This would hopefully help all future groups go through their chartering process.

**Comment by melvincarvalho** [comment]
Seems to be 2 projects, 2 names, with an overlap

1. Social project stems from Linked data and FOAF, foaf-protocols etc. and is about users, connections, and friends.  Typical apps: micro blogging, chat

2. Data storage stems from Linked Data Platform, and is about standards to read and write data with access control.  Typical apps:  document editor, file explorer.

Name should indicate whether the focus will be on chunk (1) or chunk (2) or a "big tent" approach.  Personally (1) resonates with me quite alot.  Each choice will have a few trade-offs.  Probably the easiest way to reduce the objections is to cut out the non-essential stuff, and move that into v2.

**Comment by hzbarcea** [comment]
Ready to close. I don't seem to have the necessary karma for that.
